package.json

```diff
@@ -103,6 +103,7 @@
   },
   "devDependencies": {
     "@commitlint/cli": "^19.0.3",
+    "all-contributors-cli": "^6.26.1",
```

Comment:
It seems that this dependency is not [required anymore](https://github.com/all-contributors/cli). Am I wrong?

Reply:
I've removed it.